user-guide/Getting_started/Accessing_DataMiner/Accessing_DataMiner_Cube/Installing_DataMiner_Cube/DataMiner_Cube_deployment_methods.md

```diff
@@ -29,41 +29,118 @@ Below you can find an overview of the different methods that can be used to depl
+
+    - *MainFeature*: CubeLauncher + one Cube version (mandatory).
+    - *Autorun*: Adds a registry key to show the CubeLauncher icon in the notification area (i.e. "systray") at logon for all users.
+    
+        - If enabled, users cannot disable the autorun feature.
+        - If not enabled, each user can choose to enable the autorun feature.
+
+    - *ProtocolHandler*: Adds a registry key to support the `cube://` protocol handler for all users.
+
+    Example showing how to select individual features:
+
+    ```txt
+    msiexec /i CubeVersion-10.2.1.msi ADDLOCAL=MainFeature,Autorun,ProtocolHandler
+    ```
+    
+    Example showing how to select all features:
+
+    ```txt
+    msiexec /i CubeVersion-10.2.1.msi ADDLOCAL=ALL
+    ```
+
+    Installing multiple `CubeVersion.msi` packages side-by-side will keep the latest version of DataMinerCube.exe (i.e. the launcher) in place:
+
+    - Installing an older version after a newer version will not overwrite the launcher.
+    - Uninstalling the newest version will not roll back the launcher to the previously installed version.
+
+    > [!CAUTION]
+    > Uninstalling the last installed version (regardless of whether this is a newer or older version) will remove the desktop and start menu shortcuts. To restore these shortcuts, repair one of the other installed versions by running a command like the following one: `msiexec /fs CubeVersion-10.2.1.msi`
+
 ## CefSharp MSI
 
-- Can be combined with any of the Cube deployment methods.
-- Avoids the need for each individual user to download the CefSharp web browser plugin from a DMA.
-- Deployment is in a fixed location: `C:\ProgramData\Skyline\DataMiner\DataMinerCube\CefSharp\`
-- Windows 8, 8.1 and 2012 R2 require Microsoft Visual C++ Runtime 2015 to be installed separately. For more information, see [the latest supported Visual C++ downloads](https://support.microsoft.com/en-us/topic/the-latest-supported-visual-c-downloads-2647da03-1eea-4433-9aff-95f26a218cc0).
+- Whatever Cube deployment method you use, you can install the CefSharp web browser plugin using one of the following packages.
+
+    - Use [CefSharp v81](https://community.dataminer.services/download/dataminer-cube-msi-cefsharp-v81/) to installs CefSharp v81 on clients where you installed DataMiner Cube v10.2.2 using the Cube MSI installer.
+    - Use [CefSharp v96](https://community.dataminer.services/download/dataminer-cube-msi-cefsharp-v96/) to install CefSharp v96 on clients where you installed DataMiner Cube v10.2.0 or v10.2.3 and above using the Cube MSI installer.
+
+- Installing the CefSharp web browser plugin is mandatory when you opted for the shared MSI deployment.
+
+- Installing the CefSharp web browser plugin avoids the need for each individual user to download the CefSharp web browser plugin from a DMA.
+
+- The CefSharp web browser plugin will always be installed in the following folder: `C:\ProgramData\Skyline\DataMiner\DataMinerCube\CefSharp\`
```

Comment:
This statement is no longer correct for 10.2 and can be removed.
This applied to the old v81 CefSharp MSI that was only available on-demand and was compatible with DataMiner 10.0 and 10.1 versions. As mentioned in the General MSI remarks, all MSI packages install in the same `INSTALLDIR` folder, so that will also be `C:\Program Files\Skyline Communications\DataMiner Cube\` for the CefSharp MSI.

Reply:
Corrected